FlorenceBML/BMLacq680/BMLacq680.xml

```diff
@@ -111,15 +111,14 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                     ክርስቲያን፡ የመንፈሳዊ፡ ሥራ፡ ሁሉ፡ ይፈጸምበታል፡ የቤተ፡ ክርስቲያን፡ የመንፈሳዊ፡ መንፈሳዊ፡ ሥራ፡ እንደምንድነው፡ መቀደስ፡ ማጥመቅ፡ ንሥሐ፡ የገቡትን፡ 
                                     ከኃጢኦት፡ ምፍታት፡ ምዕመናን፡ ማስተማር፡ ነው። </incipit>
                                 <explicit xml:lang="am">የዚያን፡ ጊዜ፡ ጸድቃን፡ ብርሃን፡ ትሸልመው፡ የዘለዓለምን፡ ክብር፡ ይውርሳሉ። ኃጥዓንም፡ ኃፍረት፡ ለብሰው፡ ገሃነም፡ ተፈርይባቸው፡ 
-                                    ፈቃዱን፡ በሠሩለት፡ ከሰይጣን፡ ጋራ፡ ዘለዓለም፡ ሲሣቀዮ፡ ይኖራሉ። ከዚህ፡ ሃይማኖት፡ አያናውጸን፡ አሜን። ዝመጽሐፍ፡ ዘደጃዝማች፡ 
-                                    <persName ref="PRS14025Balca" role="owner">ባልቻ</persName>፡ ወስመ፡ ጥምቀቱ፡ 
-                                    <persName ref="PRS14025Balca" role="owner">ተክለ፡ ሚካኤል፡</persName> 
-                                    ዘወጽአ፡ እምብሉይ፡ ወሐዲስ።</explicit>
+                                    ፈቃዱን፡ በሠሩለት፡ ከሰይጣን፡ ጋራ፡ ዘለዓለም፡ ሲሣቀዮ፡ ይኖራሉ። ከዚህ፡ ሃይማኖት፡ አያናውጸን፡ አሜን። ዝመጽሐፍ፡ 
+                                    ዘ<persName ref="PRS2435Balcasa" role="owner"><roleName type="title">ደጃዝማች፡</roleName> ባልቻ፡</persName> 
+                                    ወስመ፡ ጥምቀቱ፡ <persName ref="PRS2435Balcasa" role="owner">ተክለ፡ ሚካኤል፡</persName> ዘወጽአ፡ እምብሉይ፡ ወሐዲስ። </explicit>
                             </msItem>
                         </msItem>
                         <msItem xml:id="ms_i2">
                            <locus from="29ra" to="291ra"/>
-                           <title ref="LIT2375Synaxa"/>
+                            <title ref="LIT4033Senkessar"/>
```

Comment:
It's probably the "vulgate" second recension, LIT4035SenkessarDL

Reply:
OK. I will change it to the more specific ID.